## MESSAGE

# The list of doors belonging to a room
string[] doors
